pkg/controller/elasticquota.go

```diff
@@ -111,20 +111,20 @@ func (ctrl *ElasticQuotaController) Run(workers int, stopCh <-chan struct{}) {
 	defer runtime.HandleCrash()
 	defer ctrl.eqQueue.ShutDown()
 	// Start the informer factories to begin populating the informer caches
-	klog.Info("Starting Elastic Quota control loop")
+	klog.InfoS("Starting Elastic Quota control loop")
 	// Wait for the caches to be synced before starting workers
-	klog.Info("Waiting for informer caches to sync")
+	klog.InfoS("Waiting for informer caches to sync")
 	if ok := cache.WaitForCacheSync(stopCh, ctrl.eqListerSynced, ctrl.podListerSynced); !ok {
 		klog.Fatalf("Cannot sync caches")
```

Comment:
@Huang-Wei I couldn't figure out how to migrate this to `ErrorS`. Could you please help me out here? Thanks!

Reply:
You can do sth like this:

```go
klog.ErrorS(nil, "Cannot sync caches")
os.Exit(1)
```

ref:L

- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#replacing-fatal-calls
- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#replacing-fatal-calls

Reply:
Ohh okay, I didn't know that we can pass nil in case of the `err` parameter. I'll make the necessary changes!